.github/workflows/docker-test.yml

```diff
@@ -14,15 +14,30 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
-      - name: Start containers
+      - name: Install requirements
+        # make sure nvme-cli installed (we need it for /etc/nvme/hostnqn and /etc/nvme/hostid)
```

Comment:
Actually, you can use `stasadm` to create /etc/nvme/[hostnqn|hostid]`

Reply:
yep, saw it. we will need nvme-cli anyway for later tests, so decided to install it